index.html

```diff
@@ -195,113 +237,92 @@ <h2>About Myself</h2>
               </li>
             </ul>
           </div>
-          <button class="simple-button">Get my resume</button>
+          <button class="simple-button" type="button">Get my resume</button>
         </div>
-        <div class="flex">
-          <div class="frame2">
-            <ul>
-              <li>
-                <div class="lang-container">
-                  <div class="language-card">
-                    <div class="l-title"><h2>Languages</h2></div>
-                    <div class="l-icon">
-                      <i class="fa-solid fa-angle-down"></i>
-                    </div>
-                  </div>
-                  <div class="card2">
-                    <div class="frame3">
-                      <div class="img">
-                        <img src="./images/js.png" alt="" />
-                      </div>
-                      <div class="text">JavaScript</div>
-                    </div>
-                    <div class="frame3">
-                      <div class="img">
-                        <img src="./images/html.png" alt="" />
-                      </div>
-                      <div class="text">HTML</div>
-                    </div>
-                    <div class="frame3">
-                      <div class="img">
-                        <img src="./images/css.png" alt="" />
-                      </div>
-                      <div class="text">CSS</div>
-                    </div>
-                  </div>
+
+        <ul class="about-tools ul">
+          <li class="about-lang">
+            <div class="about-lang-top">
+              <div><h2>Languages</h2></div>
+              <div>
+                <i class="fa-solid fa-angle-down"></i>
+              </div>
```

Comment:
- [ ] Nice work implementing some if the changes requested by the previous reviewer, however, the colors of still need some work to look like the Figma ones 🔽 

| Your Design | Figma Design |
|--------|--------|
| ![2023-04-01_181902](https://user-images.githubusercontent.com/104319462/229307953-3fd32b25-5266-4574-a975-e1afb16ea512.png) | ![2023-04-01_181824](https://user-images.githubusercontent.com/104319462/229307958-9da6cd98-f250-49b8-b99c-c615e4fadefe.png) |

Reply:
- **[OPTIONAL]** Well done updating the text colors as requested by the previous reviewer 👏🏼, however, there is still room for improvement, the text red highlighted in the screenshots below its **`layout`** as well as its **`size`** and **`weight`** should be similar to the Figma design 👍🏼

- Currently, the right side text is not aligned correctly with the left side text.
- The font size and weight are quite larger than the one in the Figma design.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/229312318-51d1a5f6-7ae1-416f-b34e-40e3b972a06f.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/229312378-e390739c-dfbb-4e29-8f88-644b12578f4e.png)